spec/dry/types/definition_spec.rb

```diff
@@ -16,33 +16,37 @@
   end
 
   describe '#try' do
-    subject(:try) { type.try(value) }
-
-    let(:success) { try.success?    }
-    let(:failure) { try.failure?    }
-    let(:input)   { try.input       }
+    let(:result) { type.try(value) }
 
     context 'when given valid input' do
       let(:value) { 'foo' }
 
-      specify { expect(success).to be(true)  }
-      specify { expect(failure).to be(false) }
-      specify { expect(input).to be(value)   }
+      it 'returns a success' do
+        expect(result).to be_success
```

Comment:
It's unfortunate, I prefer when predicate methods return either `true` or `false`, but having said that, I don't see any good reason to be very conservative about it in tests. Mut cov remained 100% anyway :)

Specs should be focusing on behavior, and here behavior is that `if success?; puts 'ok'; else puts 'not ok'; end` works as expected, not that the return value is `true` and not `1`.